lib/components/common/CdeDetails.tsx

```diff
@@ -1,6 +1,6 @@
-import { Box, Typography, Grid, Link } from "@mui/material";
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them